### ISSUE TITLE
Modifying parameter on a team in Group wipes it without planned change

### ISSUE THREAD

**Issue by jan-ludvik** [OpsLevel/terraform-provider-opslevel#31]
We recently added Groups into Opslevel and assigned teams in them - this was done in UI.
Our terraform doesn't manage groups - it doesn't create them or assigns memberships.
However when we changed a responsibilities field of the teams, terraform removed all Groups memberships without announcing that in a terraform plan.

I would expect that - similar to aliases - if I don't have the groups field specified it will not be touched by terraform. And if it will be, at least tell us in the plan.

- If there is no change to any resource, terraform correctly reports `No changes. Your infrastructure matches the configuration.`

- If there is any change like modifiyng responsibilities:

**Terraform plan for a team:**
```
  # opslevel_team.this["aaa-bbb"] will be updated in-place
  ~ resource "opslevel_team" "this" {
        id               = "<REDACTED>"
        name             = "<REDACTED>"
      ~ responsibilities = <<-EOT
            <REDACTED>
            <REDACTED>
          + test
            Managed by terraform
        EOT
        # (4 unchanged attributes hidden)
    }
```

This plan removed this team's membership in its Group.

**Comment by rocktavious** [comment]
Sorry for leaving you hanging @jan-ludvik - We never got around to this and then the ball got dropped.  We have added this to our internal ticket tracking tool now (the rest of the company uses Gitlab issues)

We are currently working on big refactor that we suspect is causing all our nullability bugs like this so after the dust settles from that we'll circle back around on this one and ensure it gets a fix.

**Comment by rocktavious** [comment]
Hey @jan-ludvik - Sorry for taking so long to get back around to this.  I just retested this in the latest Terraform provider we are about to release and was not able to reproduce it.  I suspect this has been fixed since https://github.com/OpsLevel/terraform-provider-opslevel/releases/tag/v0.7.0 but it will for sure be fixed in `> v0.8.2` as thats what I just tested.
